src/components/DataTable/DataTable.scss

```diff
@@ -160,6 +149,8 @@ $breakpoint: 768px;
   justify-content: flex-end;
   align-items: baseline;
   transition: color duration() easing();
+  padding: spacing();
```

Comment:
What is the padding value achieving?

Reply:
This padding was on the Cell surrounding the button. In order to make the button take up the whole cell, I made the padding 0 on the Cell and moved the padding here to the button instead